test/test_helper.exs

```diff
@@ -1,27 +1,18 @@
 # remove excluded tags to run specific module tests
-ExUnit.start(exclude: [
-  :balance,
-  :boleto_log,
-  :boleto_payment_log,
-  :boleto_payment,
-  :boleto,
-  :keys,
-  :transaction,
-  :transfer_log,
-  :transfer,
-  :utility_payment_log,
-  :utility_payment,
-  :webhook,
-  :event
-])
-
-
-defmodule StarkBankTest.Credentials do
-
-  @project_id "9999999999999999"
-  @private_key "-----BEGIN EC PRIVATE KEY-----\nMHQCAQEEIBEcEJZLk/DyuXVsEjz0w4vrE7plPXhQxODvcG1Jc0WToAcGBSuBBAAK\noUQDQgAE6t4OGx1XYktOzH/7HV6FBukxq0Xs2As6oeN6re1Ttso2fwrh5BJXDq75\nmSYHeclthCRgU8zl6H1lFQ4BKZ5RCQ==\n-----END EC PRIVATE KEY-----\n"
-
-  def project() do
-    StarkBank.project(:sandbox, @project_id, @private_key)
-  end
-end
+ExUnit.start(
+  exclude: [
+    # :balance,
+    # :boleto_log,
+    # :boleto_payment_log,
+    # :boleto_payment,
+    # :boleto,
+    # :keys,
+    # :transaction,
+    # :transfer_log,
+    # :transfer,
+    # :utility_payment_log,
+    # :utility_payment,
+    # :webhook,
+    # :event
+  ]
+)
```

Comment:
Where should the project definition (id and private key) be placed for the tests to run properly?

Reply:
in `config/config.exs`.

If you need different configs for different environments, do:

```
import_config "#{Mix.env()}.exs"
```

Then you create `config/test.exs`:

```
config :starkbank,
  project: [
    environment: :sandbox,
    id: "12345",
    private_key: """
    ...
    """
  ]
```

I didn't include the secrets in the file.

Another alternative would be to include default settings in the project that read from environment variables. It would make it easier for developers to test them.

Reply:
Nice! It worked 👍